Description: The group for reading /var/log/kern.log is "adm", not "admin".
Author: Kees Cook <kees@debian.org>
Bug-Debian: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=660078

Index: apparmor-debian/utils/notify.conf
===================================================================
--- apparmor-debian.orig/utils/notify.conf	2010-11-03 17:03:52.000000000 -0700
+++ apparmor-debian/utils/notify.conf	2012-04-24 11:54:27.997521983 -0700
@@ -12,4 +12,4 @@
 show_notifications="yes"
 
 # Only people in use_group can use aa-notify
-use_group="admin"
+use_group="adm"
